lib/saveImageDiff.js

```diff
@@ -94,7 +94,7 @@ module.exports = function(imageDiff,done) {
                 misMatchPercentage: misMatchTolerance,
                 isExactSameImage: misMatchTolerance === 0,
                 isSameDimensions: imageDiff.isSameDimensions,
-                isWithinMisMatchTolerance: that.misMatchTolerance > misMatchTolerance
+                isWithinMisMatchTolerance: true
```

Comment:
It would be redundant. Take a look at the outer condition, which looks like this:
```js
if(this.misMatchTolerance < misMatchTolerance) {
    //...
    this.self.resultObject //...
        isWithinMisMatchTolerance: false
    //...
} else {
    //...
    this.self.resultObject //...
        isWithinMisMatchTolerance: true
    //...
}
```
if you'd fix it with `that.misMatchTolerance >= misMatchTolerance` the it would be:
```js
if(this.misMatchTolerance < misMatchTolerance) {
    //...
    this.self.resultObject //...
        isWithinMisMatchTolerance: false
    //...
} else {
    //...
    this.self.resultObject //...
        isWithinMisMatchTolerance: that.misMatchTolerance >= misMatchTolerance
    //...
}
```
So actually the `else` from the outer condition covers `that.misMatchTolerance >= misMatchTolerance`